chainer/training/extensions/micro_average.py

```diff
@@ -0,0 +1,72 @@
+import collections
+
+from chainer import reporter
+from chainer.training import extension
+from chainer.training import util
+
+
+class MicroAverage(extension.Extension):
```

Comment:
Hm, you're right. I now think it's good to keep it a general micro-averaging extension. Then, the head line of the docstring should be `Calculates micro-average ratio` instead of "accuracy".